inference/core/active_learning/README.md

```diff
@@ -0,0 +1,269 @@
+# Active Learning
+
+This feature is designed to facilitate the systematic collection of data from actual production distributions with 
+minimal user-side overhead, leveraging the hosting capabilities of models and datasets on the Roboflow platform.
+
+The core concept is straightforward:
+
+* The user initiates the creation of an Active Learning configuration within the Roboflow app.
```

Comment:
Is `Active Learning` a feature or the technique we are using? If is it a feature, we should use capital letters. Otherwise, we should use lowercase letters.